src/Agent.Worker/Release/ReleaseJobExtension.cs

```diff
@@ -212,9 +205,10 @@ public ReleaseJobExtension()
                 await retryExecutor.ExecuteAsync(
                     async () =>
                         {
-                            //TODO:SetAttributesToNormal
```

Comment:
https://github.com/Microsoft/vsts-agent/commit/3a2983c22d5f6cfee613f518da5d4500a2ce00bf If you notice this commit, earlier we we just calling deletdirectory. We have to reset the read permission before deleting it. We are using IOUtil.DeleteDirectory now, which takes care of this. But when I changed this forgot to remove this comment.